app/controllers/orders/client_summary.js

```diff
@@ -17,29 +21,78 @@ export default detail.extend({
 
   identityTypes: Ember.computed(function() {
     return [
-      { name: 'Hong Kong Identity Card', id: 1 },
-      { name: 'Asylum Seeker Recognizance Form', id: 2}
+      { name: "Hong Kong Identity Card", id: 1 },
+      { name: "Asylum Seeker Recognizance Form", id: 2 }
     ];
   }),
 
+  isLongDescription: Ember.computed("model.purposeDescription", function() {
+    return this.get("model.purposeDescription.length") > LONG_DESCRIPTION_SIZE;
+  }),
+
+  toggleShowMore: Ember.observer("isLongDescription", function() {
+    const isLongDescription = this.get("isLongDescription");
+    if (isLongDescription) {
```

Comment:
you could simply do `this.set("showMore", isLongDescription)` 

note: I don't believe that there's actually anything wrong with the if/else here, it's still pretty understandable and readable to me.

Reply:
In case when description is one liner, we can't show `show more` . that is why I am explicitly hiding it.

Reply:
I know, I just meant you could shorten that piece of code : )